_posts/2020-04-30-insights.adoc

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: 'Quarkus Insights on Youtube Live'
+date: 2020-04-30 
+tags: annoucement insights
```

Comment:
We are going crazy on the tags, I think we will need to calm down very quickly for them to remain discoverable

announcement
arc
awards
conference
development-tips
elasticsearch
extension
fault-tolerance
graalvm
hibernate
ide
kafka
microprofile
newsletter
performance
release
sentry
smallrye
spring
testing
tracing
user-story

Reply:
which one do you want to use? "announcement" or "insights" ? I added insights to make a llink similar to user-story but i'm not fuzzed ;)